01_prepare_host.sh

```diff
@@ -127,18 +127,18 @@ else
 fi
 
 KUBECTL_LATEST=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
-KUBECTL_LOCAL=$(kubectl version --client --short | cut -d ":" -f2 | sed 's/[[:space:]]//g' 2> /dev/null)
```

Comment:
This was not working. At some point the command started to give both the kubectl and the kustomize version.